### PR TITLE
Update async

### PR DESCRIPTION
**Pull request by eexit**
Hi,

Thank you very much for this great shell.

I was also concerned by https://github.com/mafredri/zsh-async/issues/5 so here's the PR to update async :)

Thanks,

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "pure-prompt",
-  "version": "1.3.0",
+  "version": "1.3.1",
```
Comment (sindresorhus) [inline package.json]: Don't bump. We'll do it on release.

File: package.json [hunk 2]
```diff
@@ -19,7 +19,7 @@
     "node": ">=0.10.0"
   },
   "scripts": {
-    "postinstall": "dest=/usr/local/share/zsh/site-functions; mkdir -p $dest && ln -sf \"$PWD/pure.zsh\" $dest/prompt_pure_setup && ln -sf \"$PWD/async.zsh\" $dest/async || echo 'Could not automagically symlink the prompt. Check out the readme on how to do it manually: https://github.com/sindresorhus/pure#manually'"
+    "postinstall": "dest=\"/usr/local/share/zsh/site-functions\"; mkdir -p $dest && ln -sf \"$PWD/pure.zsh\" \"$dest/prompt_pure_setup\" && ln -sf \"$PWD/async.zsh\" \"$dest/async\" || echo 'Could not automagically symlink the prompt. Check out the readme on how to do it manually: https://github.com/sindresorhus/pure#manually'"
```
Comment (sindresorhus) [inline package.json]: I don't see the point in quoting these?

Comment (eexit) [inline package.json]: When I tried to install at first, I got double slash: `/usr/local/share/zsh/site-functions//prompt_pure_setup` and it'd fail. No idea why though..

Comment (mafredri) [inline package.json]: The double slash has been fixed on master, but not released. Sorry about that!

Comment (eexit) [inline package.json]: Great, I'm removing the commit then :)

**Comment by mafredri** [comment]
I think the double quotes in post-install are unnecessary, but I'd be happy to merge the updated zsh-async. I've actually been meaning to push a new release (and update async) for a while now, but I've been waiting for a status update on #214.

**Comment by mafredri** [comment]
Thanks for the PR (and nudge)!

**Comment by eexit** [comment]
You are welcome, another one is on its way 😉

**Pull request merged**